### ISSUE TITLE
slow?

### ISSUE THREAD

**Issue by jasonkuhrt** [azz/jest-runner-tsc#5]
```
❯ npm test

> orwell@0.1.0 test /Users/jasonkuhrt/projects/dialogue/orwell
> jest src plugins

 PASS   tsc  src/server.ts (9.413s)
 PASS   tsc  plugins/hollywood.ts (9.696s)
 PASS   tsc  plugins/index.ts (10.104s)
 PASS   tsc  src/config.ts (7.157s)
 PASS   tsc  plugins/waldo.ts (7.242s)
 PASS   tsc  plugins/toktok.ts (7.062s)
 PASS   tsc  src/helpers.spec.ts
 PASS   tsc  plugins/intercom.ts
 PASS   tsc  plugins/scribe.ts
 PASS   tsc  src/poetry.ts
 PASS   tsc  src/helpers.ts
 PASS   tsc  plugins/core.ts
 PASS   tsc  plugins/datadog.ts
 PASS  src/helpers.spec.ts

Test Suites: 14 passed, 14 total
Tests:       14 passed, 14 total
Snapshots:   0 total
Time:        23.425s
Ran all test suites matching /src|plugins/i in 2 projects.

25s
```
```
❯ time npx tsc --noemit
npx tsc --noemit  7.43s user 0.37s system 136% cpu 5.726 total
```

**Comment by azz** [comment]
Wow that is slow... I never benchmarked this runner, so I don't have a baseline.

I would have expected it to be slower in total than `tsc` over a project, but not that long per-file.

**Comment by sibelius** [comment]
does `tsc` has a language server, so you could run faster?

**Comment by wzoom** [comment]
Hopefully slow builds can be solved by my PR: #14

**Comment by mmantel** [comment]
The fix was reverted in PR #16. There is significant slowness for me too: With 373 TS files, it takes about 7 minutes to run via `jest-runner-tsc`, versus 15 seconds to run `tsc` directly.
